css/selectors/blank.json

```diff
@@ -0,0 +1,60 @@
+{
+  "css": {
+    "selectors": {
+      "empty": {
+        "__compat": {
+          "description": "<code>:blank</code>",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/:blank",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": true,
+              "alternative_name": "-moz-only-whitespace",
+              "notes": "See <a href='https://bugzil.la/1106296'>bug 1106296</a>."
+            },
+            "firefox_android": {
+              "version_added": false
```

Comment:
Does `:-moz-only-whitespace` work on **Firefox for Android**? Because if so, then that should be mentioned here.